verification/curator-service/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# This dockerfile builds the curator API service and its UI in a single container.
+# The UI is served as a static resource from the curator API service express server.
+FROM node:12
+
+# Build the curator service.
+WORKDIR /usr/src/app/verification/curator-service/api
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY verification/curator-service/api/package*.json ./
+# RUN npm clean install
+RUN npm ci
+# Bundle app source
+COPY verification/curator-service/api/. .
+# Bundle common files
+COPY common/* ../../../common/
+# Build the app
+RUN npm run-script build
+# Expose service on port 3001.
+EXPOSE 3001
+
+# Build the UI
+WORKDIR /usr/src/app/verification/curator-service/ui
+# Install app dependencies
+COPY verification/curator-service/ui/package*.json ./
+# RUN npm clean install
+RUN npm ci
+# Bundle app source
+COPY verification/curator-service/ui/. .
+# Bundle common files
+COPY common/* ../../../common/
```

Comment:
nit -- if both services are in the same Dockerfile, don't need to run this twice

Reply:
good catch, removed.